tests/flytekit/unit/cli/pyflyte/test_plugin.py

```diff
@@ -0,0 +1,28 @@
+from unittest.mock import patch, Mock
+
+from flytekit.clis.sdk_in_container.plugin import get_cli_plugin, PyFlyteCLIPlugin
+
+
+@patch("flytekit.clis.sdk_in_container.plugin.entry_points")
+def test_get_plugin_default(entry_points):
+    entry_points.side_effect = lambda *args, **kwargs: []
+
+    default_plugin = get_cli_plugin()
+    assert default_plugin is PyFlyteCLIPlugin
```

Comment:
can we add a test that creates a custom CLIPlugin that modifies the `main` entrypoint via `configure_pyflyte_cli`?